app-web/plugins/gatsby-source-github-all/sourceNodes.js

```diff
@@ -38,6 +38,8 @@ const createGHNode = (file, id) => ({
   sourceName: file.metadata.sourceName, // the pretty name of the 'source'
   sourcePath: file.metadata.sourceURL, // the path to the repo
   resourcePath: file.metadata.resourcePath, // either path to a gastby created page based on this node
+  resourceTitle: file.metadata.resourceTitle,
+  resourceDescription: file.metadata.resourceDescription,
```

Comment:
Are you planning to use the description somewhere?

Reply:
yes, this is taking one step to an isolated siphon node that is not dependant on what the source type is. This is in prep for the unfurling PR